models/claims_preprocessing/claims_preprocessing_models.yml

```
@@ -1254,3 +1260,11 @@ models:
       tags: 
         - normalized_input
         - claims_preprocessing
+
+  - name: normalized_input__int_eligibility_state_normalize
+    config:
+      materialized: ephemeral
+      tags: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)